modules/rosa-deleting-cluster.adoc

```diff
@@ -32,6 +46,39 @@ $ rosa init --delete-stack
 ----
 endif::sts[]
 
+ifdef::sts[]
+.  Delete the OpenID Connect (OIDC) provider that the cluster Operators use to authenticate:
++
+[source,terminal]
+----
+$ rosa delete oidc-provider -c <cluster_id> --mode auto --yes <1>
+----
+<1> Replace `<cluster_id>` with the ID of the cluster.
+
+. Delete the cluster-specific Operator IAM roles:
++
+[source,terminal]
+----
+$ rosa delete operator-roles -c <cluster_id> --mode auto --yes <1>
+----
+<1> Replace `<cluster_id>` with the ID of the cluster.
+
+. Delete the account-wide roles:
++
+[source,terminal]
+----
+$ rosa delete account-roles --prefix <prefix> --mode auto --yes <1>
```

Comment:
Same comment as above with the `--yes` flag.

Reply:
Resolved.